themebuilder/modules/config-normalizer.js

```diff
@@ -96,6 +96,8 @@ const getThemeAndColorScheme = config => {
     };
 };
 
+const normalizePath = path => path + (path[path.length - 1] !== "/" ? "/" : "");
```

Comment:
If the path separator will be '\'?

Reply:
Just tried this and had nothing bad except the doubled "/" in the result URL.